data/json/vitamin.json

```diff
@@ -50,7 +50,7 @@
     "name": { "str": "Vitamin C" },
     "deficiency": "scurvy",
     "min": -5600,
-    "rate": "15 m",
+    "rate": "5 m",
```

Comment:
Rather too soon, I agree. There are some traits that should cause a lower starting level of various vitamins (Hates Fruit for C, Hates Meat for B12 - see below); will this require C++ coding, or can it be done via JSON?

Reply:
I'm pretty sure it would have to be C++. Traits/mutations can mess with vitamin uptake and decay rate, but AFAIK nothing JSON-able influences the vitamin storage at start. It could be a nice parallell trait, something like "Strict Vegetarian Background", which could be obligatory to some professions ( are any religious figures strictly vegetarian?) but would on its own not influence the vitamin uptake/calorie absorption. You could also then slap a harder malus onto Junky, Frail (or whichever starts you in the hospital), so that they have malnutrition effects already at start but nothing keeping them from recovering.